presto-docs/src/main/sphinx/security/internal-communication.rst

```diff
@@ -136,6 +136,7 @@ Enable JWT and set the shared secret to the same value in
 
     internal-communication.jwt.enabled=true
     internal-communication.shared-secret=<secret>
+    internal-communication.jwt.expiration-seconds=<value in seconds>
```

Comment:
@aditi-pandit, @tdcmeehan We are adding a Prestissimo only config here.
What is the current recommendation for documenting such things?

Reply:
Let's add a new section in this newly added doc: https://github.com/prestodb/presto/blob/master/presto-docs/src/main/sphinx/develop/presto-native.rst

Reply:
The thing about this page is that you need to know the path/name to browse to it. It is not listed in the "index" for the developer guide (https://prestodb.io/docs/current/develop.html).

Reply:
The lack of inclusion in the ToC is probably just an oversight and we can fix that.

Reply:
I've created PE https://github.com/prestodb/presto/pull/20792 to add the link to the develop ToC.